.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "doc/examples"]
+	path = doc/examples
+	url = https://github.com/mozilla/DeepSpeech-examples.git
+	branch = master
```

Comment:
Do we want to point to `master` ? Does that means we want to have this new repo, on `master`, in sync with the tree ? Or we just don't care about the sync ?

Reply:
It's pointing to master so that it's always up-to-date when running the examples in CI.